bootstraptutorials.html

```diff
@@ -44,6 +44,35 @@
           </div>
         </div>
       </nav>
+
+      <section class="bg-light text-dark p-5 text-center text-sm-start">
+        <div class="container">
+          <div class="d-sm-flex align-items-center justify-content-between">
+            <div>
+              <h1>
+                Becoming a Web Developer
+              </h1>
+              <p class="lead my-4">
+                Meet David and Andrew, Who are Software developers trained by microverse.
+              </p>
+              <button class="btn btn-primary btn-lg" type="submit">Learn more here</button>
+            </div>
+            <img class="img-fluid d-none d-sm-block " src="photos/Snapshoot_Portfolio_4.png" alt="">
+          </div>
```

Comment:
I think "alt" attribute should not be left empty. According to HTML and CSS best practices, It is always a good practice to assign a value to the "alt"  😊😊.